doc/method/sources.md

```diff
@@ -1,46 +1,32 @@
 ### Taxonomy sources
 
-We build the present version of OTT from eight sources. Five of these are curated online taxonomies : NCBI, GBIF, IRMNG, Index Fungorum, and WoRMS. The remaining three are an automatically generation classification based on sequence clustering (SILVA); a family-specific classification based on a published phylogeny (Lamiales); and a set of user-generated taxonomy patches.
+We build the present version of OTT from eight sources. Five of these are curated online taxonomies : NCBI, GBIF, IRMNG, Index Fungorum, and WoRMS. The remaining three are an automatically generation classification based on sequence clustering (SILVA); a family-specific classification based on a published phylogeny (Lamiales); and a set of user-generated taxonomy patches. The primary inputs are NCBI and GBIF, while incorporation of other taxonomies has been driven by curator requests.
 
-*NCBI Taxonomy.*  The first objective of the Open Tree taxonomy is to
+The choice and ranking of input taxonomies reflects the need for 1) mapping tips from phylogenies to taxa in taxonomies; 2) a backbone hierarchy in the absence of phylogenies that cover a particular clade; and 3) coverage of the synthetic tree beyond the tips present in the input trees to the [MMM] species species that are described in the literature [citation needed]. We also limit the list to taxonomies that are available for download and licensed for re-use [re-wording probably required, and decide if we want to specifically mention taxonomies that we wanted to use but can't].
+
+*NCBI Taxonomy*  
+The first objective of the Open Tree taxonomy is to
 align OTUs across phylogenetic studies.  This need is largely met by
 using the NCBI taxonomy, since (1) most phylogenies are molecular, (2)
 publishers require molecular sequences used in studies to be deposited
 in Genbank, (3) every Genbank deposit is annotated with a taxon from
-NCBI taxonomy.  NCBI also tends to be more-phylogenetically informed than other taxonomies (see Results, below), which means that it also plays a key role in providing backbone information to Open Tree of Life in the absence of phylogenetic trees for a given clade. The NCBI taxonomy therefore forms the nucleus of OTT.
+NCBI taxonomy.  NCBI also tends to be more phylogenetically-informed than other taxonomies (see Results, below), which makes it a good candidate as a default backbone. The NCBI taxonomy therefore forms the nucleus of OTT.
+
+The disadvantage of the NCBI taxonomy is that it has only [NNN] species
```

Comment:
I would not say 'disadvantage'. I would be positive and say that we have several objectives (listed in intro) and NCBI meets one of them while not meeting others.